lambdas/src/common/security/jwt-verifier.ts

```diff
@@ -20,13 +20,14 @@ export class JwtVerifier {
             const jwtVerifyOptions = this.createJwtVerifyOptions(signingAlgorithm, expectedClaimValues);
             const { payload } = await jwtVerify(encodedJwt, publicKey, jwtVerifyOptions);
 
-            if (mandatoryClaims && mandatoryClaims.size) {
-                mandatoryClaims.forEach((mandatoryClaim) => {
-                    if (!payload[mandatoryClaim]) {
-                        throw new Error(`Claims-set missing mandatory claim: ${mandatoryClaim}`);
-                    }
-                });
-            }
+            if (mandatoryClaims?.size == 0) throw new Error("No mandatory claims provided");
```

Comment:
This comparison is going to potentially be `undefined == 0` - which doesn't quite do what we expect:

```
undefined == 0
false
undefined === 0
false
```
In JavaScript it's usually better to test for [strict equality ===](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) rather than [equality ==](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality), as the latter does implicit type casting.

The previous code was checking if the value was passed through as undefined, or if it had a size property (assuming its a method), which is a very JavaScript way of defensively coding.

This is Typescript, so the property has to be a Set, which means it has a `size` method, even if that's 0. I'm not 100% sure what the behaviour is when its `undefined` - maybe try writing a test to expose that and see what happens?